data/apps/reviewer_apps.json

```diff
@@ -1,26 +1,84 @@
  },
  {"type": "packaged",
   "status": "pending",
-  "categories": ["books", "music"],
+  "name": "Pinterest",
+  "categories": ["lifestyle", "social"],
   "device_types": ["firefoxos", "desktop"],
-  "privacy_policy": "do not track",
-  "preview_files": ["images/screenshot1.png"],
-  "package_file": "manifests/app3.zip"
+  "privacy_policy": "Thank you for using Pinterest! We wrote this policy to help you understand what information we collect, how we use it, and what choices you have...",
+  "preview_files": ["images/pinterest1.png"],
+  "package_file": "manifests/pinterest.zip",
+  "description": "Pinterest is a tool to help you discover and plan things you want to do. When you find something that looks interesting, just Pin it! Pins are visual bookmarks that you organize and share the way you want.\nWith Pinterest you can:\n• Plan projects—home remodels, car restorations, garden redesigns and other DIYs\n• Dream up your next trip—outdoor adventures, road trips with friends, family outings and exotic vacations\n• Collect your favorite things—illustrations, tattoo ideas, hilarious quotes, tech gadgetry and inspiring art and architecture\n• Save great ideas—articles to read, movies to watch, gifts to buy, fashion and beauty how-tos, food and recipes to cook, fitness tips and lifehacks\n• Get organized for a party—wedding decor, birthday themes, costume ideas and game day menus"
+ },
+ {"type": "packaged",
+  "status": "pending",
+  "name": "Outlook",
+  "categories": ["books", "travel"],
+  "device_types": ["firefoxos", "desktop", "mobile"],
+  "privacy_policy": "To learn more about how Outlook protects your privacy, please visit: http://go.microsoft.com/fwlink/p/?LinkId=253457",
+  "preview_files": ["images/outlook1.png"],
+  "package_file": "manifests/outlook.zip",
+  "description": "This official Microsoft app gives you access to your email for your <a rel=\"nofollow\" href=\"http://Outlook.com\">Outlook.com</a> account. \nYou can sign in with your existing Microsoft account, which is usually your <a href=\"http://Outlook.com\">Outlook.com</a> email address and password."
+ },
+ {"type": "packaged",
+  "status": "pending",
+  "name": "CutTheRope",
+  "categories": ["games"],
+  "device_types": ["firefoxos", "desktop", "mobile"],
+  "privacy_policy": "http://www.cuttherope.net/privacy.html",
+  "preview_files": ["images/ctr1.png"],
+  "package_file": "manifests/ctr.zip",
+  "description": "Cut the rope to feed candy to little monster Om Nom®! 400 million downloads around the world of this phenomenal puzzle game. \n\nA mysterious package has arrived, and the little monster inside has only one request… CANDY! Collect gold stars, discover hidden prizes and unlock exciting new levels in this additively fun, award-winning, physics-based game!"
```

Comment:
You'll need to update some of these categories now. E.g. 'books' no longer exists.